Pipfile

```diff
@@ -7,6 +7,8 @@ verify_ssl = true
 
 [packages]
 pyyaml = "*"
+semantic_version = "*"
+attrs = "*"
```

Comment:
Please use version ranges from current version to next major version instead of using star versions (e.g. `semantic_version = ">=2.6.0,<3.0"`)